### PR TITLE
Check for impersonation cookie.

### PR DESCRIPTION
**Pull request by rkuykendall**

### CODE REVIEW

File: src/analytics/actions.es6 [hunk 1]
```diff
@@ -32,13 +32,15 @@ function isLoggedIn() {
 }
 
 function isImpersonating() {
-    const impersonating = has(global, '__env.user.real_admin_tf_login');
+    const inEnv = has(global, '__env.user.real_admin_tf_login');
+    const inCookie = has(cookies, 'Tf-Impersonating');
 
-    if (impersonating) {
+    if (inEnv || inCookie) {
         log('No analyitcs for impersonating users.');
```
Comment (learykara) [inline src/analytics/actions.es6]: `analyitcs`

**Comment by tholex** [comment]
👍 as long as `Tf-Impersonating` is the right string / cookie name, not sure where to double check that

**Comment by tati** [comment]
LGTM 👍

File: src/analytics/actions.es6 (hunk 1 above)
Comment (rkuykendall) [inline src/analytics/actions.es6]: 👍

**Comment by rkuykendall** [comment]
@tholex Double checked here https://github.com/Thinkful/hawk/pull/191 👍

**Pull request merged**